docs/src/main/asciidoc/getting-started-guide.adoc

```diff
@@ -33,8 +33,8 @@ To complete this guide, you need:
 
 * less than 15 minutes
 * an IDE
-* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
-* Apache Maven 3.5.3+
+* {jdk-version} installed with `JAVA_HOME` configured appropriately
+* {maven-version}
```

Comment:
Aren't these in mostly all the guides?

I'm thinking that maybe we should have some proper includes with JDK requirements, Maven requirements, GraalVM requirements.

They could be simple .adoc file in an `include` directory similar to what we did with https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/duration-format-note.adoc that we could move to this directory too.

Reply:
Yes, these references are most in the guides. My thought was that this was what `include::./attributes.adoc[]` was for in this doc, but maybe I am misunderstanding what you're saying. 

My confusion is that in the quarkus website repo there is an [attributes.adoc](https://github.com/quarkusio/quarkusio.github.io/blob/develop/_guides/attributes.adoc) with all the guide attributes defined. The [attributes.adoc](https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/attributes.adoc) in this repo is empty. This was why I opened the pr to the website repo and not here. 

But if there is a better approach or something I am misunderstanding, feel free to point me in the right direction.

Reply:
What I'm saying is that I think we should have include files for the common requirements.

Having the JDK version is nice but we still duplicate ``installed with `JAVA_HOME` configured appropriately`` everywhere.

So I would keep what you're doing but also create includes for the common requirements (similar to what we did with https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/duration-format-note.adoc) so that we could include each requirement when they are needed.

I can draft what I have in mind if it's not clear. And if it is, could you do it on one guide only and then, once we agree, we could generalize.